build/Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
+FROM golang:1.14.2
```

Comment:
can we use a lighter image?

Reply:
alpine doesn't seem to work as desired. We will add this in the next set of PRs.